bin/deps.ml

```diff
@@ -25,11 +25,15 @@ let run (`Setup ()) (`Syntax syntax) (`File file) =
           file;
         exit 1
   in
-  let doc = Mdx.parse_file syntax file in
-  let deps = Mdx.Dep.of_lines doc in
-  let deps = List.map Mdx.Dep.to_sexp deps in
-  Printf.printf "%s" (Mdx.Util.Sexp.Canonical.to_string (List deps));
-  0
+  match Mdx.parse_file syntax file with
```

Comment:
It might be nice to extract a function here that takes care of matching over the result, reporting the error and exiting with the right status so that we don't have to wrap everything in a pattern matching and think about this every time.

Reply:
done